schema/swagger-extensions.json

```diff
@@ -1457,6 +1457,88 @@
         }
       }
     },
+    "primitivesItemsWithFile": {
```

Comment:
Just wondering -- can this be implemented using `allOf` `primitivesItems` (so we're not copy/pasting the whole definition?) - in the event the `primitivesItems` changes in the future, it'd be nice to propagate automatically

Reply:
Yep, I can give it a shot.

Reply:
Thanks for the suggestion and figuring out how to make this work behind the scenes.  I've deduplicated the common parts into `primitivesItemsBase` so that only the enum values of the two inherited schemas will be specified.